requirements.txt

```diff
@@ -21,3 +21,5 @@ gunicorn==20.0.4
 flask-cors==3.0.10
 flask-swagger-ui==3.20.9
 PyYaml==5.4
+jinja2<3.1.0
```

Comment:
Curious why this is `<` rather than `==` or `>`.

Reply:
It's `<` and not `>` because there was a breaking API change after this version.

As for why it's not `==`... it might as well be! I'll change that.